examples/fall_detection/demo.ipynb

```diff
@@ -0,0 +1,354 @@
+    "To store the final output, person's pose, and mid-product, list of keypoints, we create two properties in `Person` VObj.\n",
+    "\n",
+    "Since ST-GCN requires keypoints be stored for the last 30 frames, function that computes `keypoints` needs to be decorated with `@stateful(30)`, where `30` specifies that 30 frames of values should be saved.\n",
+    "\n",
+    "Adding the two properties to `Person`, we have:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class Person(vqpy.VObjBase):\n",
+    "    required_fields = ['class_id', 'tlbr']\n",
+    "\n",
+    "    @vqpy.property()\n",
+    "    @vqpy.stateful(30)  # require 30 frames of \n",
+    "    def keypoints(self):\n",
+    "        image = self._ctx.frame\n",
+    "        tlbr = self.getv('tlbr')\n",
+    "        # per-frame property, tlbr could be None when tracking is lost\n",
+    "        # temporary work around until we have better dependency control\n",
+    "        if tlbr is None:\n",
+    "            return None\n",
+    "        return pose_model.predict(image, torch.tensor([tlbr]))\n",
+    "\n",
+    "    @vqpy.property()\n",
+    "    def pose(self) -> str:\n",
+    "        keypoints_list = []\n",
+    "        # retrieve list of keypoints from the last 30 frames\n",
+    "        # also need to deal with object lost during tracking\n",
+    "        # return 'unknown' if not enough keypoints\n",
+    "        for i in range(-self._track_length, 0):\n",
+    "            keypoint = self.getv('keypoints', i)\n",
+    "            if keypoint is not None:\n",
+    "                keypoints_list.append(keypoint)\n",
+    "            if len(keypoints_list) >= 30:\n",
+    "                break\n",
+    "        if len(keypoints_list) < 30:\n",
+    "            return 'unknown'\n",
+    "        # type conversion to adapt data to model input\n",
+    "        pts = np.array(keypoints_list, dtype=np.float32)\n",
+    "        out = action_model.predict(pts, self._ctx.frame.shape[:2])\n",
+    "        action_name = action_model.class_names[out[0].argmax()]\n",
+    "        return action_name"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "> The conditional statements `if tlbr is None` in L11 and iteration over `for i in range(-self._track_length, 0)` in L21 are attempting to get the most recent bounding box and keypoints. We are still working on more convenient ways to access properties (and stateful properties) in VObjs."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 2: Query on `Person`'s pose\n",
+    "\n",
+    "To filter on people that are falling down, we filter on `pose` having value `\"Fall Down\"` (7 actions should be supported: `\"Standing\", \"Walking\", \"Sitting\", \"Lying Down\", \"Stand up\", \"Sit down\", \"Fall Down\"`).\n",
```

Comment:
> (7 actions should be supported: `\"Standing\", \"Walking\", \"Sitting\", \"Lying Down\", \"Stand up\", \"Sit down\", \"Fall Down\"`)
move to the explanation of `pose` property

Reply:
Moved